### DISCUSSION TITLE
Translation issue

### DISCUSSION THREAD

**Discussion by SammyTheNerdRage** [iSoron/uhabits#1013] [Technical Discussion] [answered]
Hi, in polish language notification says "Czy wyrwałeś dziś w nawyku?" instead of "Czy wytrwałeś dziś w nawyku?" (Have you persevered in the habit today?).
![Screenshot_20210709-152155_Telegram](https://user-images.githubusercontent.com/86573942/125084103-702d5700-e0c9-11eb-8331-6216e58a8997.jpg)

"Wyrwałeś" correspond to pull something from something else e.g. some vegetables from the ground or door from jamb.
"Wytrwałeś" correspond to do some activity despite the adversity.

Can you insert "t" into "wyrwałeś"?
Thanks.

**Comment by hiqua** [comment, marked as answer]
Feel free to correct the issue there: https://translate.loophabits.org/
